easybuild/easyconfigs/v/Vim/Vim-8.0-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,36 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'ConfigureMake'
+
+name = 'Vim'
+version = '8.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.vim.org'
+description = """ Vim is an advanced text editor that seeks to provide the power 
+ of the de-facto Unix editor 'Vi', with a more complete feature set. """
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['http://ftp.vim.org/pub/vim/unix/']
+sources = [SOURCELOWER_TAR_BZ2]
+
+# this dependencies are required for --enable-pythoninterp=yes and --enable-perlinterp=yes to work
+# check enabled features running "vim --version" after compilation
+dependencies = [
+    ('Python', '2.7.11'),
+    ('Perl', '5.22.1', '-bare')
+]
+#osdependencies = ['python','perl-devel']
```

Comment:
@RvDijk please remove this, serves no purpose since these are both listed in `dependencies`?